### PR TITLE
[CMDCT-3412, CMDCT-3416]: Fix console warnings

### PR DESCRIPTION
**Pull request by braxex**
### Description

We have several pesky console warnings that appear both in-app and when running unit tests.

### Related ticket(s)

CMDCT-3412, CMDCT-3416

---
### How to test

Run the following unit tests and observe that the associated console errors do not appear: 
1. `DashboardPage.test.tsx`:  
    - _Warning: Encountered two children with the same key, `mock-report-id`._
2. `ExportedEntityDetailsTable.test.tsx`: 
    - _Warning: validateDOMNesting(...): `<p>` cannot appear as a descendant of `<p>`._
    - _Warning: React does not recognize the `showHintText` prop on a DOM element._
3. `ExportedReportWrapper.test.tsx`:
    - _Warning: Each child in a list should have a unique "key" prop._

### Notes

n/a

---
### Pre-review checklist

- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [x] I have updated relevant documentation, if necessary
- [x] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by gmrabian — DISMISSED** [review]
removes the warnings; deployed env looks ok

**Review by jessabean — COMMENTED** [review]
Pointing out a couple of CSS issues on the export page due to the markup changes:

- Missing some space between the section with the cards and the title of the next section (not sure exactly which exported modal entity is which here, but I think one is the `ExportedEntityDetailsOverlay` and one is the `ExportedModalOverlay` ... maybe ... 😆 )
- Spacing between the entity status icon and the headings is a bit tight

Screenshot 2024-04-22 at 9 14 12 AM

Screenshot 2024-04-22 at 9 36 04 AM

**Comment by braxex** [comment]
@jessabean @britt-mo I wasn't sure where to look for designs for that padding but assumed some styling would look weird when I unnested those tables and was counting on design to call it out lol. Happy to change it if someone will point me in the right direction.

File: services/ui-src/src/components/export/ExportedOverlayModalReportSection.tsx [hunk 1]
```diff
@@ -32,12 +32,7 @@ export const ExportedOverlayModalReportSection = ({
     }
   );
   return (
-    <Box
-      as="td"
-      mt="2rem"
-      data-testid="exportedOverlayModalPage"
-      sx={sx.container}
-    >
+    <Box mt="2rem" data-testid="exportedOverlayModalPage" sx={sx.container}>
```
Comment (britt-mo) [inline services/ui-src/src/components/export/ExportedOverlayModalReportSection.tsx:35]: https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=2413-99200&mode=design&t=NjwIz8mBkNnfCMfF-0

This is the mockup for the export

**Comment by braxex** [comment]
@jessabean @britt-mo Fixed the styling issues. @benmartin-coforma Will you double-check your recently merged work you did with Jess (https://github.com/Enterprise-CMCS/macpro-mdct-mfp/commit/dce17487e13279e783ba6fdbcf1e08fa1e597c94) to make sure my change of that whole area from table to non-table didn't mess anything up? Specifically looking at the closed out initiative part in `services/ui-src/src/components/export/ExportedModalOverlayReportSection.tsx`.

**Review by jessabean — APPROVED** [review]

**Review by benmartin-coforma — APPROVED** [review]
I don't see any issues in the code, and I trust Brax on the styling. @jessabean: we should follow up and remove `section` from `ExportedEntityDetailsOverlaySectionProps`; it's never used.

**Pull request merged**